--- lalpulsar-6.1.0/lib/PSDutils.c.orig	2024-06-09 09:06:21.881832033 +0200
+++ lalpulsar-6.1.0/lib/PSDutils.c	2024-06-09 09:10:57.030341426 +0200
@@ -562,7 +562,7 @@ REAL8 XLALMathOpOverArray( const REAL8 *
     ; /* empty statement because declaration cannot be first line in a switch case */
     size_t *sortidx;
     if ( ( sortidx = XLALMalloc( length * sizeof( size_t ) ) ) == NULL ) {
-      XLALPrintError( "XLALMalloc(%ld) failed.\n", length );
+      XLALPrintError( "XLALMalloc(%zd) failed.\n", length );
       XLAL_ERROR( XLAL_ENOMEM );
     }
     gsl_sort_index( sortidx, data, 1, length );
