--- lalpulsar-6.1.0/Makefile.am.orig	2024-06-09 08:48:17.267707892 +0200
+++ lalpulsar-6.1.0/Makefile.am	2024-06-09 09:06:02.015272993 +0200
@@ -26,11 +26,6 @@ user_environment = \
 	trailingcolon MANPATH; \
 	set $(PACKAGE)_PREFIX $(prefix); \
 	set $(PACKAGE)_DATADIR $(pkgdatadir); \
-	prepend MANPATH $(mandir); \
-	prepend OCTAVE_PATH $(octexecdir); \
-	prepend PATH $(bindir); \
-	prepend PKG_CONFIG_PATH $(libdir)/pkgconfig; \
-	prepend PYTHONPATH $(pyexecdir) $(pythondir); \
 	$(END_OF_LIST)
 
 pkgconfigdir = $(libdir)/pkgconfig
